src/api/entity/idea.ts

```diff
@@ -0,0 +1,62 @@
+import { Entity, PrimaryGeneratedColumn, Column } from "typeorm";
+
+import { SolutionType } from "../../libs/types";
+
+@Entity()
+export class Idea {
+  @PrimaryGeneratedColumn("uuid")
+  id: string;
+
+  @Column("uuid")
+  schoolId: string;
+
+  @Column("uuid")
+  teacherId: string;
+
+  @Column("text")
+  solutionName: string;
+
+  @Column("char", { length: 6 })
+  solutionType: SolutionType;
+
+  @Column("varchar", { length: 255 })
+  problemArea: string;
+
+  @Column("text")
+  problemSelection: string;
+
+  @Column("text")
+  problemReasoning: string;
+
+  @Column("text")
+  solutionVision: string;
+
+  @Column("text")
+  solutionMission: string;
+
+  @Column("text")
+  solutionBenefit: string;
+
+  @Column("text")
+  solutionObstacle: string;
+
+  @Column("varchar", { length: 255 })
+  solutionPitchUrl: string;
```

Comment:
Yg ini just keep in mind.  Url yg kita support hanya url dari youtube ya, jadi gak bisa pake url dari platform video yg lain. Mungkin perlu tambah validator urlnya.